source/read-write-concern/tests/operation/default-write-concern-2.6.yml

```diff
@@ -0,0 +1,231 @@
+            insert: *collection_name
+            documents:
+              - {_id: 3}
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: *collection_name
+            updates: 
+              - {q: {_id: 1}, u: {$set: {x: 3}}, multi: true}
+            writeConcern: null
+      - command_started_event:
+          command:
+            delete: *collection_name
+            deletes: [{q: {_id: 3}, limit: 1}]
+            writeConcern: null
+  - description: 'InsertOne and InsertMany omit default write concern'
+    operations:
+      - name: insertOne
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          document: {_id: 3}
+      - name: insertMany
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          documents:
+            - {_id: 4}
+            - {_id: 5}
+    outcome:
+      collection:
+        name: *collection_name
+        data:
+          - {_id: 1, x: 11}
+          - {_id: 2, x: 22}
+          - {_id: 3}
+          - {_id: 4}
+          - {_id: 5}
+    expectations:
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - {_id: 3}
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - {_id: 4}
+              - {_id: 5}
+            writeConcern: null
+  - description: 'UpdateOne, UpdateMany, and ReplaceOne omit default write concern'
+    operations:
+      - name: updateOne
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 1}
+          update: {$set: {x: 1}}
+      - name: updateMany
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 2}
+          update: {$set: {x: 2}}
+      - name: replaceOne
+        object: collection
+        collectionOptions: {writeConcern: {}}
+        arguments:
+          filter: {_id: 2}
+          replacement: {x: 3}
+    outcome:
+      collection:
+        name: *collection_name
+        data:
+          - {_id: 1, x: 1}
+          - {_id: 2, x: 3}
+    expectations:
+      - command_started_event:
+          command:
+            update: *collection_name
+            updates:
+              - {q: {_id: 1}, u: {$set: {x: 1}}}
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: *collection_name
+            updates:
+              - {q: {_id: 2}, u: {$set: {x: 2}}, multi: true}
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: *collection_name
+            updates:
+              - {q: {_id: 2}, u: {x: 3}}
+            writeConcern: null
+  - description: DeleteOne with write concern unset omits default write concern
```

Comment:
Why does this "with write concern unset omits default write concern" test only exist for DeleteOne? Is it leftover from a previous PR revision and something that should be deleted, or did you just arbitrarily decide to test this for one operation? I noted that the findAndModify test only does this for findOneAndUpdate, so perhaps it's the latter.

I'm wondering if this might be a fragile test if a test runner decides to use some environment variable for its MongoClient (in the absence of `clientOptions` in the test itself), which could specify its own write concern. Perhaps that's not a concern since the [Transactions test runner](https://github.com/mongodb/specifications/tree/master/source/transactions/tests) instructs using a fresh MongoClient for each test.

Reply:
It is the latter. Originally the tests were all like this, with no write concern specified on client/database/collection. In our later discussions, we decided to switch to testing with an explicit write concern set on a client/database/collection. In the slack discussion between the three of us, I proposed we still have one test of without an explicit default write concern set. Though, like the tests of a write concern unset on the client/database/collection, I think this test is not very valuable. Removed.